Wikipedia/Code/WMFArticleNavigationController.m

```diff
@@ -19,7 +19,7 @@ - (void)viewDidLoad {
     [super setDelegate:self];
 
     self.secondToolbarHidden = YES;
-    self.secondToolbar = [[UIToolbar alloc] initWithFrame:CGRectZero];
+//    self.secondToolbar = [[UIToolbar alloc] initWithFrame:CGRectZero];
```

Comment:
Looks like we don't need this second toolbar?

If not we can get rid of all it's layout methods too... i.e. `layoutSecondToolbarForViewBounds` and others in this file...